### ISSUE TITLE
Ask to overwrite the savegame if the name differs.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#215]
```
qasur stated here:
http://andors.techby2guys.com/viewtopic.php?f=4&t=844

Can we get an "Are you sure?" button when you're about to overwrite an existing 
saved file?

It should ask you if the character name being saved differs from the saved game 
you're about to erase.
```

Original issue reported on code.google.com by `SamuelPl...@gmail.com` on 9 Jul 2011 at 7:22

**Comment by GoogleCodeExporter** [comment]
```
ejwessel has started working on this for v0.6.10 .
```

Original comment by `oskar.wi...@gmail.com` on 23 Jul 2011 at 1:21
- Changed state: **Started**
- Added labels: **Type-Enhancement**, **Priority-Medium**, **Milestone-0.6.10**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Completed in r142 thanks to ejwessel.
```

Original comment by `oskar.wi...@gmail.com` on 24 Jul 2011 at 11:27
- Changed state: **Ready**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.10 . Closing.
```

Original comment by `oskar.wi...@gmail.com` on 21 Oct 2011 at 8:56
- Changed state: **Fixed**
